meson.build

```diff
@@ -1,7 +1,7 @@
 project(
   'rauc-hawkbit-updater',
   'c',
-  version : '1.2',
+  version : '1.3',
```

Comment:
So far I've updated this with the release commit, but should be ok to do it here, too.

Reply:
I just followed docs/release-checklist.txt.